build-utils/language-c/src/main/java/org/apache/plc4x/language/c/CLanguageTemplateHelper.java

```diff
@@ -436,8 +476,45 @@ private String toExpression(Field field, Term term, Function<Term, String> varia
         }
     }
 
-    public String toVariableParseExpression(Field field, Term term, Argument[] parserArguments) {
+    public String toVariableParseExpression(ComplexTypeDefinition baseType, Field field, Term term, Argument[] parserArguments) {
         VariableLiteral vl = (VariableLiteral) term;
```

Comment:
magic string

Reply:
??

Reply:
"CAST" could be a constant.  If there are other things you could do that too.

Reply:
Yeah ... correct ... but I think there's only one place where it's used ...